CHANGES.rst

```diff
@@ -32,6 +32,16 @@ Breaking changes:
 
 New features:
 
+- Add compatibility to :rfc:`9253`:
+  
+  - Add new property types :class:`icalender.vUid` and :class:`icalender.vXmlReference`.
+  - Add properties to all components: :attr:`Component.concepts`, :py:attr:`Component.links`, :py:attr:`Component.refids`, :py:attr:`Component.related_to`.
```

Comment:
All these should have consistent markup and link to the relevant attr. Is there an example of valid reST markup for `attr`?